sqlmesh/core/context.py

```diff
@@ -1846,15 +1848,77 @@ def create_external_models(self, strict: bool = False) -> None:
                 strict=strict,
             )
 
+    def get_sensitive_fields(self) -> Set[str]:
+        """Returns a set of common sensitive field names"""
+        return {
+            "password",
+            "secret",
+            "key",
+            "token",
+            "api_key",
+            "private_key",
+            "access_token",
+            "refresh_token",
+            "auth_token",
+            "credentials",
+            "client_secret",
+            "certificate",
+            "ssh_key",
+        }
+
+    def is_sensitive_field(self, field_name: str, sensitive_fields: Set[str]) -> bool:
+        """
+        Check if a field name contains any sensitive keywords
+        """
+        field_lower = field_name.lower()
+        return any(sensitive in field_lower for sensitive in sensitive_fields)
+
+    def mask_sensitive_value(self, value: Any) -> str:
+        """
+        Mask sensitive values with a placeholder
+        Returns '****' for non-empty values and '' for empty ones
+        """
+        if value and str(value).strip():
+            return "*" * len(str(value))
+        return "None"
+
+    def print_config(self, config: ConnectionConfig | None, console: Any) -> None:
```

Comment:
There's already `self` available, why pass `console` as an argument? Also let's make this private (prefix with `_`). We should be careful about adding public methods to the Context since it's a gateway into SQLMesh Python API.

Reply:
Could also be a candidate for utils

Reply:
Ok, do you prefer that I create a dedicated utils file for that?